packages/firestore/src/local/simple_db.ts

```diff
@@ -273,7 +273,16 @@ export class SimpleDb {
         objectStores
       );
       try {
+        // TODO(schmidt-sebastian): Remove this code/comment or find a way to
```

Comment:
I left this here for now to make my development process easier. This actually caught an issue during development, when I marked `getNewDocumentChanges` as idempotent. It is not yet since it modifies internal state that is used during execution.